### ISSUE TITLE
Button does not have a name

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#702]
**Describe the bug**
``` 
Error: This element has role of "button" but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.                                                                                                                                   
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.Name
   ├── #main > header > div:nth-child(1) > div:nth-child(1) > div > div:nth-child(1) > a:nth-child(4)
   └── <a href="#" class="d-inline-block d-lg-none py-2 text-white" data-activeid="slimDropdown" role="button" data-activate-element=""> <svg class="icon i...</a>
```

We should probably add a name to the element.
Tested with pa11y

**Comment by Trustidiong** [comment]
Hey, I would like to work on this issue. I think it will be fun.

**Comment by mishrajiharsh219** [comment]
Addressing #702
Hey, I would love to work on this, I think it will be fun for me.
open-source is awesome.

**Comment by libremente** [comment]
Thanks, for your interest. Assigning @mishrajiharsh219 to this one.

**Comment by bfabio** [comment]
Hi @mishrajiharsh219, are you still working on this?

**Comment by mishrajiharsh219** [comment]
No you can work on it if assigned
